starter_code/src/movies.js

```diff
@@ -1,20 +1,126 @@
 /* eslint no-restricted-globals: 'off' */
 // Turn duration of the movies from hours to minutes 
+function turnHoursToMinutes(moviesArray){
+    return  moviesArray.map((eachMovie)=>{
+        let newMovie = {...eachMovie}; // spread 
+        let regexHours = /[0-9]+h/g; //regex to find hours
+        let regexMinutes = /([0-9]+min)/g; //regex to find minutes
 
+        let hoursArray = newMovie.duration.toString().match(regexHours); //find hours
+        let minutesArray = newMovie.duration.toString().match(regexMinutes); //find minutes
+        
+        let durationInMin = 0;
+        
+        if(hoursArray !== null){
+            let hours = parseInt(hoursArray[0].replace(/\D/g, "")); //remove the h in hours
```

Comment:
I like the way you went about solving this, but just a reminder that you can parseInt() a string if it starts with a number. For example, parseInt("44minutes") will give you 44. Might let you cut some steps out, but otherwise what you did is totally correct.